### ISSUE TITLE
Do not mix with other ember-power-select

### ISSUE THREAD

**Issue by huberts** [cibernox/ember-power-select-typeahead#47]
Hi!
It seems I found an error when I try to mix your typeahead component with classic ember-power-select (single choice) component.
When they both sit on the page, when you click on the single choice component, your typeahead activates every time. The problem is with the class "ember-power-select-search-input" in your trigger, which is queried when one clicks on single choice component - and then focused. Compare:
[EPST code](https://github.com/cibernox/ember-power-select-typeahead/blob/master/addon/templates/components/power-select-typeahead/trigger.hbs#L4)
[EPS code](https://github.com/cibernox/ember-power-select/blob/master/addon/components/power-select/before-options.js#L42)

**Comment by cibernox** [comment]
Yep, that is a bug indeed. The input of the power-select should have a unique ID and the code in EPS should focus by ID instead of focusing the first thing with that class. That was a naive approach.

Can you PR a fix with some test? I'm extremely short of time this week.

**Comment by heat** [comment]
As a component you could use `this.$()` so you'll  work only in component instance.
http://emberjs.com/api/classes/Ember.Component.html#method__

**Comment by cibernox** [comment]
No, you can't on tagless components. We need an id.

**Comment by snewcomer** [comment]
I believe this issue is closed, no?
